generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.js

```diff
@@ -10,28 +10,35 @@ const DEFAULT_PUBLIC_ROUTE = Routes.LOGIN;
 const DEFAULT_PRIVATE_ROUTE = Routes.HOME;
 
 function AuthenticatedRoute({
-  // TODO Add this if you need it
-  // device,
+  /*
+   * TODO Add this if you need it
+   * device,
+   */
   isPublicRoute,
   isPrivateRoute,
   initialized,
-  currentUser,
+  // CurrentUser,
```

Comment:
Mmm 🤔

Reply:
There's a linter rule that forces all comments to start with capital case. So if I comment this, it capitalizes the user. I commented it because it isn't being used.